embedded-graphics/src/pixel_iterator.rs

```diff
@@ -26,3 +26,28 @@ where
         target.draw_iter(self)
     }
 }
+
+///  TODO: Doc
+pub trait IntoPixels<C>
```

Comment:
I just realized that if would probably better to use an associated type for `C` similar to `IntoIterator::Item`.